modules/runner/src/testFixtures/java/org/apache/ignite/internal/ClusterPerClassIntegrationTest.java

```diff
@@ -184,6 +194,44 @@ protected String getNodeBootstrapConfigTemplate() {
         return NODE_BOOTSTRAP_CFG_TEMPLATE;
     }
 
+    /**
+     * Waits for all partitions in the specified zone to reach the HEALTHY state across all cluster nodes.
+     *
+     * @param zone The name of the distribution zone to check.
+     * @param tableName The name of the table (currently unused by the implementation, reserved for future use).
+     * @param partitionsCount The number of partitions to check (currently unused by the implementation, reserved for future use).
+     * @throws InterruptedException If the thread is interrupted while waiting.
+     * @throws AssertionError If partitions do not become healthy within the timeout period.
+     */
+    protected void awaitPartitionToBeHealthy(String zone, String tableName, int partitionsCount) throws InterruptedException {
```

Comment:
- awaitPartition**s**ToBeHealthy
- `partitionsCount` naming is misleading, it's rather `Set<Integer> partitionIds`